### PR TITLE
Add: Head Title Everywhere

### PR DESCRIPTION
**Pull request by firefinchdev**

### CODE REVIEW

**Comment by firefinchdev** [comment]
@abhishek97 Pls Review #179

File: app/index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta name="msvalidate.01" content="FAD3370B879C516ED95DC8FDCCB3537C" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>CodingblocksOnline</title>
+    <!--<title>CodingblocksOnline</title>-->
```
Comment (abhishek97) [inline app/index.html]: remove it altogether.

File: app/pods/application/route.js [hunk 2]
```diff
@@ -75,5 +76,8 @@ export default Route.extend(ApplicationRouteMixin, {
       later(function(){
         controller.set('code', undefined)
       })
+    },
+    afterModel(model) {
+      this.set('headData.title', 'Coding Blocks Online')
```
Comment (abhishek97) [inline app/pods/application/route.js]: Remove space between Coding and Blocks.

**Comment by abhishek97** [comment]
@firefinchdev Look good. Do the required changes please

File: app/pods/application/route.js (hunk 2 above)
Comment (witty123) [reply app/pods/application/route.js]: Let the space be there, we can target three different key words,
1. Coding
2. Blocks
3. Coding Blocks

**Comment by firefinchdev** [comment]
Done @abhishek97

**Pull request merged**